src/game.cpp

```diff
@@ -1541,6 +1541,9 @@ bool game::do_turn()
     u.process_turn();
     if( u.moves < 0 && get_option<bool>( "FORCE_REDRAW" ) ) {
         draw();
+#ifdef TILES
```

Comment:
Agree, might be simplest to rename it to "refresh_display" or something and have it a no-op in non-tiles build.